src/frontend/src/components/GMMatrixVisualization2/GMMatrixVisualization2.js

```diff
@@ -142,7 +142,7 @@ export default class GMMatrixVisualization2 extends PureComponent {
         .then(res => {
             res.json()
             .then (json => {
-                //console.log(json)
+                // console.log(json)
```

Comment:
please remove commented-out code in this file

Reply:
I have deleted this code.